src/apiClients/AirTableApiClient.ts

```diff
@@ -68,6 +88,19 @@ class AirTableApiClient {
     this.cachedOrganizations = new CachedItem<Organization[]>(1000);
   }
 
+  /**
+   * Returns a list of organizations by making an API call to AirTable.
+   * Uses the API's select method, which makes a GET request to the Organizations endpoint.
+   * Takes in optional filters to retrieve organizations with specific fields.
+   *
+   * @param filters - Object containing lists of options to filter by. If these lists are empty,
+   * no organizations will be filtered out. Within each list, organizations only need to satisfy at least one
+   * of the desired options. For example, if multiple interest categories are specified, an organization only needs
+   * to match one of them to be returned. Between the lists, an organization must satisfy at least one option from each.
+   * Therefore, an organization would need to have at least one of the desired interest categories AND one of the desired river sections
+   * to prevent being filtered out.
+   * @returns An array of Organization objects satisfying the desired filters
+   */
```

Comment:
We should consider a maximum characters per line (even if our linter doesn't have this). However, the priority is for the docs to look nice so the generated HTML takes first priority. Something to look into...
@teresaliu20